.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+- package-ecosystem: npm
+  directory: "/"
```

Comment:
Are you sure this actually works? There's no package.json in the root, does this find all packages or actually just do nothing? (I suspect the latter.)

Reply:
Good call, played around locally, and added the 3 specific paths with the package.json files